crates/trigger/src/cli/stdio.rs

```diff
@@ -86,6 +95,23 @@ impl StdioLoggingExecutorHooks {
             _ => Ok(()),
         }
     }
+
+    fn truncate_log_files(&self, log_dir: &Path) {
+        if let Ok(entries) = log_dir.read_dir() {
+            for entry in entries.flatten() {
+                let path = entry.path();
+                let Some(name) = path.file_name().and_then(|n| n.to_str()) else {
+                    continue;
+                };
+
+                if name.ends_with(&format!("{STDOUT_LOG_FILE_SUFFIX}.txt"))
+                    || name.ends_with(&format!("{STDERR_LOG_FILE_SUFFIX}.txt"))
+                {
+                    _ = std::fs::File::create(log_dir.join(name))
```

Comment:
Could this use the `path` variable (set on line 102) instead of regenerating the path from a `join`?